test/labeling/model/test_label_model.py

```diff
@@ -95,7 +95,9 @@ def test_generate_O(self):
                 [1 / 4, 0, 0, 1 / 4, 0, 1 / 4],
             ]
         )
-        np.testing.assert_array_almost_equal(label_model.O.numpy(), true_O)
+        np.testing.assert_array_almost_equal(
+            label_model.O.cpu().detach().numpy(), true_O
```

Comment:
Feels very strange to see `label_model.O', but I know that's not part of this PR's scope.